packages/@o3r/rules-engine/builders/rules-engine-extractor/helpers/rules-engine.extractor.ts

```diff
@@ -45,26 +61,27 @@ export class RulesEngineExtractor {
   private readonly commentParser = new ConfigDocParser();
 
   constructor(tsconfigPath: string, private readonly basePath: string, private readonly logger: LoggerApi) {
-    const {config} = ts.readConfigFile(tsconfigPath, ts.sys.readFile);
+    // eslint-disable-next-line @typescript-eslint/unbound-method -- parameter of `readConfigFile` is of type `(path: string) => string | undefined`
+    const { config } = ts.readConfigFile(tsconfigPath, ts.sys.readFile);
     this.tsconfig = config;
   }
 
   /**
    * Extract type definition of a type reference into schema file
-   *
    * @param type Name of the type to extract
    * @param sourceFile path to the code source file
    */
   private async extractTypeRef(type: string, sourceFile: string) {
-    const internalLibFiles = this.tsconfig.extraOptions?.otterSubModuleRefs?.length > 0 ?
-      await lastValueFrom(zip(...(this.tsconfig.extraOptions?.otterSubModuleRefs as string[]).map((value: string) => globby(value))).pipe(
+    const internalLibFiles = this.tsconfig.extraOptions?.otterSubModuleRefs?.length > 0
+      ? await lastValueFrom(zip(...(this.tsconfig.extraOptions?.otterSubModuleRefs as string[]).map((value: string) => globby(value))).pipe(
```

Comment:
It does not complain on the line length?

Reply:
141 and max value is 200 so it seems okay